### ISSUE TITLE
remove data validator from simorgh

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#3697]
**Is your feature request related to a problem? Please describe.**
The data validator is no longer being run as part of the unit tests, the PR https://github.com/bbc/simorgh/pull/3374/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L21 removed the dataValidate command and removed it being used in the test command. This means the data validation logic which used to validate the article fixture payloads isn’t being triggered any more and has become redundant. 

https://github.com/bbc/articles-validator
^ is another schema which is a combined effort of the articles, ares and optimo team to make a shared schema. This uses [AJV](https://www.npmjs.com/package/ajv) to validate the payload. In the long term we want Ares to use this schema generated by `articles-validator` to validate the payloads in production before serving them to all downstream renderers. 

**Describe the solution you'd like**
We should remove the `dataValidator` logic from simorgh. While we await Ares to integrate the validation logic into production we should use the `articles-validator` schema along with `ajv` in the deploy pipeline to validate our article fixture data. 

**Describe alternatives you've considered**
Remove the `dataValidator` logic, but not integrate the `articles-validator` logic therefore not validating the article fixture data. This would need to be confirmed with the articles pod. 

**Testing notes**
[Tester to complete]

Dev insight: No testing required. 

- [ ] This feature is expected to need manual testing.

**Additional context**
Members of the Core pod have previously mentioned that validate static data all the time seems excessive, however this is irrelevant if the validation logic happens quickly (EG: less than a few seconds). Alternatively the validation could be run periodically instead, either automated or manually by the articles pod.

**Comment by dr3** [comment]
@pjlee11 Where do you want us to move the validator to?

**Comment by pjlee11** [comment]
Updated to `remove validator`. Ideally we should integrate with the new `articles-validator` (most likely in the pipeline rather than local dev) and therefore delete `dataValidator` from Simorgh

**Comment by thekp** [comment]
remove this folder: `simorgh/dataValidator`
and we will create a new issue to add the new article-validator in simorgh 3.0
make sure all reference of dataValidator is 🛀 cleaned

**Comment by oluoluoxenfree** [comment]
Blocked by #3853

**Comment by sareh** [comment]
Closed as fixed in https://github.com/bbc/simorgh/pull/4589
